3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -297,30 +296,121 @@ Nella parte inferiore del pannello sono presenti i pulsanti contenenti le scritt
+) <icona_lista_ordini>
+
+
+=== Richiesta spostamento prodotto <richiesta_spostamento_prodotto>
 
-Alla sua pressione verrà reso disponibile, sulla destra della schermata, il pannello "Nuova zona" (@immagine_pannello_nuova_zona) nel quale sarà possibile inserire tutti i dati utili alla creazione della zona personalizzata.
+Lo spostamento di un prodotto tra due bin avviene tramite _drag and drop_:
+
+  - posizionando il puntatore del mouse sul bin contenente il prodotto che si desidera spostare e premendo il tasto sinistro del mouse, è possibile "prendere" il prodotto;
+
+  - tenendo premuto il medesimo tasto è possibile spostare il prodotto nella posizione del bin di destinazione;
+
+  - rilasciando il tasto, verrà visualizzata nella parte inferiore destra dello schermo una notifica che avviserà dell'avvenuto, o meno, ordine di spostamento.
+
+
+Nel caso in cui l'ordine di movimentazione sia avvenuto correttamente, i bin di partenza e di arrivo verranno evidenziati rispettivamente in giallo e verde (@immagine_spostamento_prodotti).
 
 #figure(
-  image("./imgs/pannello_nuova_zona.png", width: 50%),
+  image("./imgs/spostamento_drag_and_drop.png", width: 40%),
   caption: [
-    Pannello di creazione di una nuova zona
+    Visualizzazione spostamento prodotti nella zona
   ],
-) <immagine_pannello_nuova_zona>
+) <immagine_spostamento_prodotti>
+
+\
+I prodotti non collocati, visualizzabili nell'apposita sezione del menù *Prodotti* (@visualizzazione_lista_prodotti), possono essere posizionati tramite il pannello di ispezione del bin di destinazione (@ispezione_bin), purché esso sia vuoto, ma tale operazione non comporta la generazione di una richiesta di spostamento.
+
+
+== Ricerca zone <ricerca_zone>
 
+Nella parte superiore del pannello relativo alla visualizzazione della lista delle zone (@icona_lista_zone (destra)), è presente l'area di ricerca delle zone tramite `ID`.
+
+Dopo aver inserito il parametro desiderato nella barra di ricerca, verranno mostrati nell'area sottostante i risultati relativi alle zone aventi un `ID` che contenga al suo interno il parametro immesso.
+
+== Ricerca prodotti <ricerca_prodotti>
+
+Nella parte superiore del pannello relativo alla visualizzazione delle liste dei prodotti (@icona_lista_prodotti (destra)), è presente l'area di ricerca dei prodotti.
+
+I criteri di ricerca disponibili sono:
+
+  - mediante il nome del prodotto o una sua sottostringa. Deve essere selezionata la voce "Nome" a sinistra della barra di ricerca;
+
+  - mediante l'`ID` del prodotto o una sua sottostringa. Deve essere selezionata la voce "ID" a sinistra della barra di ricerca;
+
+  - in aggiunta ai criteri precedentemente elencati, è possibile filtrare i risultati delle ricerche in base alle categorie merceologiche. Dopo aver premuto su "Categoria prodotto", la categoria scelta deve essere selezionata dalla sottostante lista delle merci presenti nel database.
```

Comment:
Cambierei in:

"- categoria merceologica: dopo aver premuto su "Categoria prodotto" comparirà una lista nella quale è possibile selezionare la categoria desiderata."